lib/avo/fields/tags_field.rb

```diff
@@ -16,6 +17,7 @@ def initialize(id, **args, &block)
         add_array_prop args, :disallowed
         add_array_prop args, :delimiters, [","]
         add_array_prop args, :suggestions
+        add_string_prop args, :suggestions_max_items
```

Comment:
It's technically not a string, but there is no "add_integer_prop" method and it is working with `add_string_prop`.